mod/state-transition/pkg/core/state_processor.go

```diff
@@ -472,85 +470,6 @@ func (sp *StateProcessor[
 	return st.SetLatestBlockHeader(lbh)
 }
 
-// getAttestationDeltas as defined in the Ethereum 2.0 specification.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_attestation_deltas
-//
-//nolint:lll
-func (sp *StateProcessor[
-	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, _, _, _, _, _,
-]) getAttestationDeltas(
```

Comment:
the main point is dropping this function, which does really nothing. Other removals follows from thia